tests/bluetooth/controller/ctrl_cte_req/src/main.c

```diff
@@ -616,6 +616,751 @@ void test_cte_req_reject_inv_ll_param_peripheral_remote(void)
 				  "Free CTX buffers %d", ctx_buffers_free());
 }
 
+/* Tests related with PHY update procedure and CTE request procedure "collision" */
+
+#define PREFER_S2_CODING 0U
+#define HOST_INITIATED 1U
+#define PHY_UPDATE_INSTANT_DELTA 6
+
+#define CHECK_PREF_PHY_STATE(_conn, _tx, _rx)                                                      \
+	do {                                                                                       \
+		zassert_equal(_conn.phy_pref_rx, _rx,                                              \
+			      "Preferred RX PHY mismatch %d (actual) != %d (expected)",            \
+			      _conn.phy_pref_rx, _rx);                                             \
+		zassert_equal(_conn.phy_pref_tx, _tx,                                              \
+			      "Preferred TX PHY mismatch %d (actual) != %d (expected)",            \
+			      _conn.phy_pref_tx, _tx);                                             \
+	} while (0)
+
+#define CHECK_CURRENT_PHY_STATE(_conn, _tx, _flags, _rx)                                           \
+	do {                                                                                       \
+		zassert_equal(_conn.lll.phy_rx, _rx,                                               \
+			      "Current RX PHY mismatch %d (actual) != %d (expected)",              \
+			      _conn.lll.phy_rx, _rx);                                              \
+		zassert_equal(_conn.lll.phy_tx, _tx,                                               \
+			      "Current TX PHY mismatch %d (actual) != %d (expected)",              \
+			      _conn.lll.phy_tx, _tx);                                              \
+		zassert_equal(_conn.lll.phy_rx, _rx,                                               \
+			      "Current Flags mismatch %d (actual) != %d (expected)",               \
+			      _conn.lll.phy_flags, _flags);                                        \
+	} while (0)
+
+static bool is_instant_reached(struct ll_conn *conn, uint16_t instant)
+{
+	return ((event_counter(conn) - instant) & 0xFFFF) <= 0x7FFF;
+}
+
+static uint16_t pu_event_counter(struct ll_conn *conn)
+{
+	struct lll_conn *lll;
+	uint16_t event_counter;
+
+	/* TODO(thoh): Lazy hardcoded */
+	uint16_t lazy = 0;
```

Comment:
What is this used for?

Reply:
The variable is not used. It was related with peripheral latency but it is not needed in this set of unit tests. Removed.